docs/source/io_formats/settings.rst

```diff
@@ -398,6 +398,33 @@ or sub-elements and can be set to either "false" or "true".
 
   .. note:: This element is not used in the multi-group :ref:`energy_mode`.
 
+------------------------
+``<random_ray>`` Element
+------------------------
+
+The ``<random_ray>`` element enables random ray mode and contains
+a number of settings relevant to the solver. Tips for selecting these parameters
+can be found in the :ref:`random ray user guide <random_ray>`.
+
+  :distance_inactive:
+    The inactive ray length (dead zone length).
+
+    *Default*: None
+
+  :distance_active:
+    The active ray length.
+
+    *Default*: None
+
+  :source:
+    Specifies the starting ray distribution, and follows the format for
```

Comment:
So let me get this straight: for fixed source mode, the rays are actually sampled according to the source distribution? In particular, shouldn't this allow an angular distribution, spatial distribution, and multigroup energy distribution?

This is opposed to eigenvalue mode where the goal is to cover space uniformly.

Reply:
The ray source does not represent any sort of physical neutron/photon particle source. It is going to be independent of any fission/scattering/fixed sources or what the simulated particle flux looks like regardless of if solving in eigenvalue or fixed source mode. The ray source is more akin to an integration quadrature. It's the same idea as if sampling some unknown function `f(x)` with Monte Carlo sampling. You just pick random values for x to test and see what `f(x)` is at that point and keep track of it. The average value of `f(x)` after many random samples of `x` is the value of the integral. If you happened to know that the value of `f(x)` is much higher for some values of `x` than others, that doesn't mean you can only sample those points -- otherwise you will be left with a biased integral. If you have a way of weighting to maintain a fair game that's another thing, but absent some sort of weighting scheme you just have to sample points randomly. So too with random rays, so we just sample rays randomly from a uniform distribution in space and angle, even if running in fixed source. I.e., the sampling of rays and their paths is independent of the particle flux/source of the simulation problem.

In fixed source mode (upcoming in the next PR) with random ray, the user specifies fixed sources more or less in the same manner as with MC (as well as specifying the ray source). The fixed particle sources need to be volumetric sources with isotropic angular distributions and with a discrete energy distribution, which are then preprocessed to generate volumetric source terms for each FSR.